Makefile

```diff
@@ -74,6 +74,7 @@ down: ## Tears down the flask and harvester containers
 clean: ## Cleans docker images
 	docker compose down -v --remove-orphans
 	docker compose -p harvest-app down -v --remove-orphans
+	sleep 5
```

Comment:
have you tried adding a custom `sleep5` command and then shimming that in between `up` and `load-test-data`